analyses/sample-distribution-analysis/02-multilayer-pie.R

```diff
@@ -0,0 +1,91 @@
+# This script creates a treemap and a multilayer pie chart to represent the
+# broad histologies, short histologies, and molecular subtypes within the dataset.
+#
+# This script uses the packages sunburstR, d3r, and treemap to produce the
+# visualizations.
+#
+# Chante Bethell for CCDL 2019
+
+# Load/install packages. This will be removed once a Dockerfile is established.
+source("00-install-packages.R", print.eval = FALSE)
+
+# Load source script
+source("01-filter-across-types.R")
+
+# Set path to plots directory
+outputDir <- file.path("/plots")
+
+# magrittr pipe
+`%>%` <- dplyr::`%>%`
+
+# Create a colorblind-friendly color vector
+color <- colorblindr::palette_OkabeIto
+
+sun_df <- df2 %>%
+  dplyr::select(broad_histology, short_histology, disease_type_new) %>%
```

Comment:
Since you're doing this selection here, you can probably use `complete.cases` in conjunction with `dplyr::filter` instead of `!is.na` 3 times. In addition, I _think_ you'd be able to put all three of those calls in the same `dplyr::filter` call even if you stuck with `!is.na` 3x but there may be some funkiness I'm not aware of without running through this myself.